### PR TITLE
added support for LXN59-2S7LX1.0 switch by 3A Smart Home DE

### PR DESCRIPTION
**Pull request by lostpoint-ru**
added support for the switch I just got from aliexpress.
no new from/to handlers, used existing ones.

### CODE REVIEW

**Comment by lostpoint-ru** [comment]
Link to device: https://aliexpress.ru/item/4001093548153.html

`{"id":10,"type":"Router","ieeeAddr":"0x086bd7fffe3f8c37","nwkAddr":58239,"manufId":4478,"manufName":"3A Smart Home DE","powerSource":"Unknown","modelId":"LXN59-2S7LX1.0","epList":[1,2],"endpoints":{"1":{"profId":260,"epId":1,"devId":256,"inClusterList":[0,3,4,5,6],"outClusterList":[],"clusters":{"genBasic":{"attributes":{"modelId":"LXN59-2S7LX1.0","manufacturerName":"3A Smart Home DE","powerSource":0,"zclVersion":3,"appVersion":0,"stackVersion":0,"hwVersion":0,"dateCode":"20200609"}},"genOnOff":{"attributes":{"onOff":0}}},"binds":[]},"2":{"profId":260,"epId":2,"devId":256,"inClusterList":[0,3,4,5,6],"outClusterList":[],"clusters":{"genBasic":{"attributes":{"appVersion":0,"stackVersion":0,"hwVersion":0}},"genOnOff":{"attributes":{"onOff":0}}},"binds":[]}},"appVersion":0,"stackVersion":0,"hwVersion":0,"dateCode":"20200609","zclVersion":3,"interviewCompleted":true,"meta":{},"lastSeen":1594668598250}`

File: devices.js [hunk 1]
```diff
@@ -371,6 +371,20 @@ const livolo = {
 };
 
 const devices = [
+    // 3A Smart Home DE
+    {
+        zigbeeModel: ['LXN59-2S7LX1.0'],
+        model: 'LXN59-2S7LX1.0',
+        vendor: '3A Smart Home DE',
```
Comment (Koenkk) [inline devices.js:378]: https://aliexpress.ru/item/4001093548153.html shows this is a Zemismart device which doesn't not match the vendor here, what is the correct one?

**Comment by Koenkk** [comment]
Note to self: after this merge https://github.com/Koenkk/zigbee2mqtt/pull/3905/

File: devices.js (hunk 1 above)
Comment (lostpoint-ru) [reply devices.js:378]: I bought it from exact link above, and it shows `"manufId":4478,"manufName":"3A Smart Home DE"` in database.db.

I have only 1-gang version btw.

Comment (lostpoint-ru) [reply devices.js:378]: Also, the box was very generic, without any brand or logo on it.

Comment (Koenkk) [reply devices.js]: Thanks, added Zemismart as the whitelabe.

**Pull request merged**